src/debian/8.2/coredeps/Dockerfile

```diff
@@ -1,18 +1,19 @@
 FROM debian:8.2
 
-# Install tools used by the VSO build automation.  nodejs-legacy is a Debian specific
-# package that provides `node' on the path (which azure cli needs).
+# Install tools used by the VSO build automation. Set up a new apt-get source to
+# get a new version of node and npm: the built-in old cert is no longer valid.
 RUN apt-get update \
+    && apt-get install -y wget \
+    && wget -O - https://deb.nodesource.com/setup_8.x | bash \
     && apt-get install -y \
-            git \
-            nodejs \
-            nodejs-legacy \
-            npm \
-            tar \
-            zip \
+        git \
+        zip \
```

Comment:
Would you mind alphabetizing these to follow the Dockerfile best practices?

Reply:
Sure, sorted where I copied it from too.